rmdir/src/rmdir.yml

```diff
@@ -0,0 +1,20 @@
+name: rmdir 
+version: "0.0.0"
+author: Giacomo Parolini <silverweed1991@gmail.com>
+about: "Remove empty directories\n\n"
+args:
+    - DIRECTORY:
+        help: The directory(ies) to be removed
+        required: true
+        multiple: true
+    - ignore-fail-nonempty:
+        long: ignore-fail-on-nonempty
+        help: Ignore each failure that is solely because a directory is non-empty
```

Comment:
We have a rule to always have a short and a long.

Create a short for this one, since either the implementation doesn't have this option or don't have a short, you're free to choose one for this one :smiley: